src/Serializer/DataArraySerializer.php

```diff
@@ -46,6 +46,6 @@ public function item($resourceKey, array $data)
      */
     public function null()
     {
-        return ['data' => []];
+        return [$resourceKey ?: 'data' => []];
```

Comment:
ah, right. That would make sense. Who would be the right person to ask about collection() and item() ?